docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -136,6 +136,52 @@ Using a custom namespace solves problems later on because you do not have to ove
 
 1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
 
+## Generate some test metrics
+
+{{< docs/shared source="alloy" lookup="agent-deprecation.md" version="next" >}}
+
+The Grafana Mimir Helm chart can collect metrics, logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
+In the example that follows, metamonitoring scrapes metrics about Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
+
+1. Download the Grafana Agent Operator Custom Resource Definitions (CRDs) from
+   https://github.com/grafana/agent/tree/main/operations/agent-static-operator/crds.
+
```

Comment:
In itself this sentence implies that the user download those files one-by-one, which is a PITA. 
Maybe we should just reference https://grafana.com/docs/agent/latest/operator/getting-started/#deploy-the-agent-operator-custom-resource-definitions-crds  in step 1 and remove step 2, since we do have the link checking in the website to protect against this link breaking.

Alternatively, let's tell the user to download the repo contents via https://github.com/grafana/agent/archive/refs/heads/main.zip  , unzip it and then executing
```
kubectl create -f agent-main/operations/agent-static-operator/crds/
```

Reply:
I think we can go with the first option for simplicity. I don't love cross-referencing deprecated docs here, but we explicitly state that Agent is deprecated, and until the Helm chart is updated, I'm not sure what else we can do.